app-sec-groups.html.md.erb

```diff
@@ -90,6 +94,22 @@ Cloud Foundry applies the rules that this ASG defines to all
 application containers in the space.
 A space may belong to more than one application security group.
 
+## <a id='unbinding-groups'></a>Unbinding Application Security Groups ##
+
+<p class='note'><strong>Note</strong>: Unbinding an ASG will not affect containers that are already running. To apply changed security rules immediately, restart all running apps in the relevant space, or across your whole installation.</p>
```

Comment:
~~restart all running apps in the relevant space, or across your whole installation.~~

restart all running applications that use the relevant application security group.